content/news/2022-03-galaxy-release-22-01/index.md

```diff
@@ -0,0 +1,70 @@
+---
+title: 'Galaxy Release 22.01'
+tease: Colour Selector, Robust File Uploads, Beta History Improvements, FastAPI, Galaxy Vault & much more!
+date: '2021-11-02'
+---
+
+Dear Community,
+
+The Galaxy Committers team is pleased to announce the release of Galaxy 22.01.
+
+- **[Developer and admin release announcement](https://docs.galaxyproject.org/en/master/releases/22.01_announce.html)** 
+- **[User release announcement](https://docs.galaxyproject.org/en/master/releases/22.01_announce_user.html)**.
+
+A few release highlights are:
+
+### New Colour Selector
+
+Do you use tools which require colour inputs, like Circos? Previously we
+had a very restricted colour input which gave you a very limited
+palette. Now you have complete freedom of choice with the modern colour
+selector.
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/l2DSGWFEzyM" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+### Improved File Export
+
+If you've been exporting files from Galaxy lately, you've probably
+seen the amazing new remote file source export, which allows you to
+export files to FTP, Dropbox, and other locations.
+
+<iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/0zxux0AmS2w" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+### Improved File Uploads
+
+Previously Galaxy servers used a variety of methods to let you upload
+large files easily, including some servers which required FTP for large
+files. We have replaced this with a new upload method, which will be enabled on all usegalaxy servers soon.
+This will make file uploads significantly smoother, and will be more tolerant of network
+failures and interruptions! You do not need to make any changes.
+
+### Beta History: Collection Improvements
+
+If you've been trying out the beta history (which will be the default
+history in the next release!), it has been updated to indicate whether collections
+are homogeneous or heterogeneous, which will help you see more easily if
+you've accidentally included an incorrect dataset. Test out the beta
+history today and let us know if you have any issues!
```

Comment:
do we have a link where people can report the issues?

Reply:
```suggestion
history today and let us know if you have any issues at https://github.com/galaxyproject/galaxy/issues!
```
I don't know how much we actually want to ask for feedback given that this has already significantly changed in dev (what do you think @dannon @guerler ?) and is going to change even more.

Reply:
Maybe we should instead say people should check test.galaxyproject.org which can run a recent dev version, but the server is down currently ... so 🤷

Reply:
Yeah, it's completely different from what's in dev.  I don't know that we want to solicit more feedback on what's there.